src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.Windows.props

```diff
@@ -86,6 +86,12 @@ The .NET Foundation licenses this file to you under the MIT license.
       <LinkerArg Condition="'$(ControlFlowGuard)' == 'Guard'" Include="/guard:cf" />
     </ItemGroup>
 
+    <ItemGroup Condition="!Exists('$(IlcSdkPath)debugucrt.txt')">
```

Comment:
Why is the file necessary? I'd imagine we can propagate a property through the build system instead of writing semaphore files on disk.

Reply:
There are two builds at play here:
- The dotnet/runtime repo build. `build.cmd` at the root of this repo.
- The NativeAOT build that ships as part of the NativeAOT NuGet package. `dotnet public /p:PublishAot=true`

We need to propagate the setting from the first build to the second build. It needs to go through a file. The two builds run at different times on different machines.

Reply:
Ah. This is a sentinel in the SDK itself to indicate the behavior. I like this approach then.